common/txmgr/broadcaster.go

```diff
@@ -792,13 +797,26 @@ func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) save
 }
 
 // Used to get the next usable sequence for a transaction
-func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) GetNextSequence(address ADDR) (seq SEQ, err error) {
+func (eb *Broadcaster[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) GetNextSequence(ctx context.Context, address ADDR) (seq SEQ, err error) {
 	eb.sequenceLock.Lock()
 	defer eb.sequenceLock.Unlock()
 	// Get next sequence from map
 	seq, exists := eb.nextSequenceMap[address]
 	if !exists {
-		return seq, errors.New(fmt.Sprint("address not found in next sequence map: ", address))
+		eb.logger.Criticalw("address not found in local next sequence map. Attempting to search and populate sequence.", "address", address.String())
```

Comment:
Doesn't need to be a critical error. It just means that we couldn't fetch sequence from onchain due to an unavailable RPC.
Maybe just an info log.

Reply:
Will do! It does also mean that we can't broadcast any transaction for that address since we don't have a nonce for it